RePoE/parser/util.py

```diff
@@ -80,3 +80,19 @@ def ignore_mod_domain(domain):
         MOD_DOMAIN.AFFLICTION_JEWEL,
     }
     return domain not in whitelist
+
+
+def find_missing_stat_descriptions(file_system, data_path, relational_reader, translation_file_cache, ot_file_cache):
+    node = file_system.build_directory()
+    keys = list(node["Metadata"]["StatDescriptions"].children.keys())
+
+    missing_keys = []
+    for key in keys:
+        if "descriptions.txt" in key:
+            if key not in [written_file[0] for written_file in WRITTEN_FILES]:
+                missing_keys.append(key)
+    return missing_keys
```

Comment:
While I don't know if there is a better way to do this with PyPoE, below suggestion looks more "pythonic" to me.

```suggestion
    node = file_system.build_directory()
    keys = node["Metadata"]["StatDescriptions"].children.keys()
    written_game_files = [written_game_file for written_game_file, _ in WRITTEN_FILES]
    return [key for key in keys if "descriptions.txt" in key and key not in written_game_files]
```